src/Microsoft.AspNet.Razor/Parser/TagHelpers/TagHelperBlockRewriter.cs

```diff
@@ -104,6 +107,11 @@ public static class TagHelperBlockRewriter
 
                 if (afterEquals)
                 {
+                    // We've captured all leading whitespace, the attribute name, and an equals with an optional 
+                    // quote/double quote. We're now at: " asp-for='|...'" or " asp-for=|..."
+                    // The goal here is to capture all symbols until the end of the attribute. Note this will not 
+                    // consume an ending quote due to the symbolOffset.
```

Comment:
agree this method handles a single attribute.  however the content of `htmlSymbols` isn't obvious.  might just need a comment near where it's defined that mentions it may contain preceding whitespace, will always contain the attribute name, will likely contain equals and a value, and will not contain appended whitespace.  or a shorter comment subsetting that info :smile: